sources/packages/web/src/services/http/ApplicationApi.ts

```diff
@@ -42,4 +42,17 @@ export class ApplicationApi extends HttpBaseClient {
       throw error;
     }
   }
+
+  public async confirmationOfAssessment(applicationId: number): Promise<void> {
```

Comment:
Can we have this method renamed to confirmAssessment? It seems to be more aligned with the others and with the API call.